_data/milestones.yml

```diff
@@ -11,7 +11,7 @@
       status: Done
       url: https://github.com/uswds/uswds/issues/2296
     - title: Better typographic flexibility and resilience, with or without webfonts
-      status: In progress
+      status: Done
       url: https://github.com/uswds/uswds/issues/2295
     - title: Clearer, more specific design and implementation guidance
       status: In progress
```

Comment:
I think we can call this one done. I mean, it'll never be _done_ done but we did clean it up and connect it to tokens, etc.

Reply:
Done in 74ff4d4.